domain/src/test/java/org/oppia/domain/topic/TopicControllerTest.kt

```diff
@@ -623,6 +627,15 @@ class TopicControllerTest {
     assertThat(conceptCardLiveData.value!!.isFailure()).isTrue()
   }
 
+  @Test
+  fun testGetReviewCard_fractionSubtopicId1_isSuccessful() {
+    val reviewCardLiveData = topicController.getReviewCard(FRACTIONS_TOPIC_ID, "1")
```

Comment:
I suggest moving the ``"1"`` here to its own test constant ID like we do for the other JSON files (just for simplicity--I'm sure you'll want to reference this later for UI tests).

Reply:
Defined constant for Subtopic Id.